task-list/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1><%= @person.name %>'s Tasks</h1>
+<table id="person_tasks_table">
+  <th id="name_header">
+    Task Name
+  </th>
+  <th id="status_header">
+    Status
+  </th>
+  <% ordered_tasks = @person.tasks.order(:comp_date) %>
```

Comment:
I think it might be best to set the `@ordered_tasks` variable in the controller. While it makes sense that the ordering of a person's tasks is a view/display decision, the way that we're getting the ordered set of tasks here is based on ActiveRecord and the database.
For that reason I would recommend keeping it in the controller, or alternatively using the `sort_by` method instead of `order`.